### PR TITLE
Bagxml

### LINKED ISSUE #5: Add schema for bagxml
Add a schema for the the bagxml seen for entries in a coda site instance at APP/bag/.

### PR DESCRIPTION
**Pull request by somexpert**
This should temporarily fix #5.

### CODE REVIEW

**Comment by somexpert** [comment]
@ldko, @damonkelley, this is ready for review. Not much to look at, but I'm thinking I might have some things that need to be changed in the README. Might have described some elements incorrectly.

File: bagxml/README.md [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+This readme, the schema, and the example document are all still a
+work-in-progress. Major changes to all the three documents are expected
+to take place.
+
+
+About
+-----
+
+This schema defines what the UNT Libraries considers to be a valid bagxml
+instance. A bagxml instance is an XML representation of a bag.
```
Comment (ldko) [inline bagxml/README.md]: We should mention that the bag that this is a representation of is a bag as defined by BagIt specification.

File: bagxml/README.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+
+
+### payloadSize ###
+
+The total size of the bag payload.
+
+
+### bagitVersion ###
+
+The version of the bagIt specs being used.
```
Comment (ldko) [inline bagxml/README.md]: specs -> specification

File: bagxml/README.md (hunk 1 above)
Comment (somexpert) [inline bagxml/README.md]: Right, I'll add that along with a link to the specifications

**Comment by somexpert** [comment]
Added the link

**Comment by somexpert** [comment]
@damonkelley, @ldko, how does this look now?

**Comment by ldko** [comment]
The schema looks fine to me. We should wait for a verdict from @vphill on which fields should be required.

**Comment by vphill** [comment]
I think it is fine to go with just lastStatus and baggingDate as optional with the other elements being required.  If we run into problems in the future with this arrangement we can adjust our opinion then.

**Comment by damonkelley** [comment]
:+1:

**Comment by ldko** [comment]
After further communication with Mark, we are going to make all fields required. Also, we will add validation of a "pass" or "fail" value being all that is accepted into the schema for lastStatus.

File: bagxml/README.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+
+### bagInfo ###
+
+The bagInfo element can contain any number of item child elements, which contain
+metadata about the bag itself.
+
+
+#### item ####
+
+An individual metadata 'item'. Must contain the child elements name and body. This
```
Comment (ldko) [inline bagxml/README.md]: Let's set off "name" and "body" with markup or quotes or something. Other than that :+1: .

**Comment by ldko** [comment]
:+1:

**Pull request merged**